lib/buffer.js

```diff
@@ -1211,14 +1211,50 @@ if (internalBinding('config').hasIntl) {
   };
 }
 
+let DOMException;
+
+const lazyInvalidCharError = hideStackFrames((message, name) => {
+  if (DOMException === undefined)
+    DOMException = internalBinding('messaging').DOMException;
```

Comment:
why do we always have to lazily load this constructor?

Reply:
Yeah, it's gotten a bit out of hand really. I'm doing it for consistency. I recently landed a PR that updated the various `AbortError` instances to eliminate this in multiple places but there are still more. What would be good is a PR that moves DOMException into `internal/errors' and replaces these lazy loads for that.